openassessment/xblock/static/js/src/studio/oa_tiny_mce.js

```diff
@@ -23,30 +23,57 @@ export const oaTinyMCE = (options) => {
 
   const getFonts = () => CUSTOM_FONTS + STANDARD_FONTS;
   const baseAssetUrl = options.base_asset_url;
+  const staticUrl = '/static';
+
+  /**
+   * This method is used to rewrite url that was copied from studio.
+   * Within Studio -> File & Uploads -> Copy Urls.
+   * If the user used copy studios, the link get will be formatted
+   * similiar to `/static/....`. This will be invalid for usage and we
+   * need to modify it to be a correct link.
+   * @param {string} content HTML String
+   * @returns {string} new content
+   */
+  const rewriteContentUrl = (content) => {
+    // These url are generated by WYSIWYG editor, it is assumed to be formatted
+    // <a href='.....' >...</a> for link.
```

Comment:
I think a future reader of this code would be able to read the lines below it and know that this code would only work for things formatted as in the example in this comment. So they can probably figure out what your comment tells them, even without you having commented it.

The part that is most non-obvious about this code is why the assumption that anchor tags (and image tags, below) can safely be assumed to be in that format. Why that is is probably the most important thing to include in a comment here.